web/src/stream_list.ts

```diff
@@ -429,13 +429,13 @@ export function zoom_in_topics(options: {stream_id: number | undefined}): void {
         const stream_id = options.stream_id;
 
         if (stream_id_for_elt($elt) === stream_id) {
-            $elt.show();
+            $elt.toggleClass("hide", false);
```

Comment:
There are two other uses of `hide()` above, but they'll both be removed soon so it didn't feel worth converting them and testing that.

Reply:
They're not the same element above, they're other elements that will be removed soon.

Reply:
`.streams_subheader` and `.stream-filters-label` specifically. If they weren't about to be removed, I'd have converted them here too for completion.

Reply:
Yeah, that looks correct. Thanks @evykassirer for pointing that out.